pkg/orchestrator/config.go

```diff
@@ -179,6 +179,9 @@ func LoadConfig() (*Config, error) {
 		providerKind = models.AWS
 	}
 
+	// TODO this is a bug since it is being resolved to "apiserver" on all of our providers installations.
```

Comment:
Not sure it's clear what the bug is in this context. Do you mean that APIServerHost is populated incorrectly by installers, or there is a bug in the code?

Reply:
I mean the APIServerHost is populated incorrectly by installers

Reply:
I am planning to address this in the PR which aims for refactoring the Orchestrator config to make sure that Orchestrator has a dedicated config option for API server host instead of reusing the config option for API server itself.

Reply:
Resolving this discussion so we can merge this PR.